libs/db-browser/src/main/resources/schema/sql/oboracle/oboracle_4_1_x.yaml

```diff
@@ -0,0 +1,277 @@
+sqls:
```

Comment:
Does this column only exists in OB 4.1.0? Is it an Oracle-compatibility feature?

Reply:
The USER_GENERATED field is only exists in ob versions 4.1.0 and above. It is a feature of the ob oracle kernel. Oracle does not have this field.